faw/pdf-observatory/requirements.txt

```diff
@@ -2,6 +2,7 @@ bokeh >= 2.1.1
 cachetools
 click==7.*
 dask[complete] == 2022.1.0
+graphlib-backport==1.0.3  # Backport of py39+ graphlib stdlib module
```

Comment:
Pinned to a patch version since this package is not official. A longer-term option is to update python.